spec/ht_members_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "ht_members"
+require "dotenv"
+
+RSpec.describe HTMembers do
+  let(:mock_umich) {
+    {
+      "umich" => HTMembers.template("us", 1.33, "EYM")
+    }
+  }
+
+  describe "mocking" do
+    it "allows you to send in mock data through new({...})" do
+      htm = HTMembers.new(mock_umich)
+      expect(htm.mocked).to be(true)
```

Comment:
`mocked`/`canned` also seem to maybe be leftovers from development -- which is to say their main purpose seems to be to support this and similar tests. Those tests seem to be primarily for the purpose of making sure that as you're writing tests the tests pass or fail for the reasons you expect -- but now that you've verified that, I think those tests aren't continuing to do a lot of work and can be removed.